distribution/resources/bin/gateway.bat

```diff
@@ -112,7 +112,7 @@ IF EXIST %CONF_OUT_FILE% DEL /Q /F %CONF_OUT_FILE%
     SET usage_path=%GW_HOME%\api-usage-data
     CALL SET USAGE_DATA_PATH=%%usage_path:\=%separator%%%
 
-    CALL :startGateway %*
+    CALL :setJavaArgs %*
```

Comment:
How about renaming this as `setupAndRun` because this label setup the java args and start the gateway
```suggestion
    CALL :setupAndRun %*
```